src/OpenIddict.Core/Views/Shared/Authorize.cshtml

```diff
@@ -1,14 +1,13 @@
 ﻿@using AspNet.Security.OpenIdConnect.Extensions
 @using Microsoft.IdentityModel.Protocols.OpenIdConnect
-@using OpenIddict.Models
 
-@model Tuple<OpenIdConnectMessage, Application>
+@model Tuple<OpenIdConnectMessage, string>
```

Comment:
since we are embedding everything I think we should create a specific ViewModel instead of passing in a Tuple.